spdx-utils/src/main/kotlin/model/SpdxExternalDocumentReference.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2017-2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.spdx.model
+
+/**
+ * [SpdxExternalDocumentReference]s are used by [SpdxDocument]s to list all external documents which are referenced from
+ * that particular [SpdxDocument], see also
+ * https://github.com/spdx/spdx-spec/blob/master/chapters/2-document-creation-information.md#26-external-document-references-.
+ */
+data class SpdxExternalDocumentReference(
+    /**
+     * The identifier referencing the external [SpdxDocument] in the format: "DocumentRef-${id-string}"
+     */
+    val externalDocumentId: String = "",
```

Comment:
Why do we use an empty string as the default if we below require this property to be set (to a non-blank value)? Shouldn't we then use no default value at all to enforce the user to set one?

Reply:
That was just by mistake.